### DISCUSSION TITLE
Набор тестеров

### DISCUSSION THREAD

**Discussion by rtx2014** [getlantern/lantern-forum-ru#12] [General]
Спасибо за обратную связь! Да, нам еще есть над чем работать. Поэтому мы открыли набор тестеров, которые готовы помочь сделать Lantern еще быстрее, стабильнее и удобнее.

Каждый тестер бесплатно получит подписку на Lantern Pro и доступ к закрытому чату с разработчиками. Вместе мы сможем быстрее обновлять Lantern, чтобы каждый пользователь в России мог не беспокоиться о доступе к нужным сайтам и сервисам.

Чтобы стать тестером, нужно отправить письмо на zima@getlantern.org и рассказать о себе.

Мы будем оперативно делиться результатами здесь. Следите за новостями!

**Comment by Esteviredzu** [comment]
Здравствуйте, а что входит в обязанности тестера?

**Comment by rtx2014** [comment]
Я просто продублировал их сообщение в канале: https://t.me/lantern_russia
